presto-main/src/main/java/com/facebook/presto/testing/TestingMetadata.java

```diff
@@ -210,6 +217,59 @@ public void dropView(ConnectorSession session, SchemaTableName viewName)
         return map.build();
     }
 
+    @Override
+    public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, Optional<ConnectorNewTableLayout> layout)
+    {
+        createTable(session, tableMetadata);
+        return TestingHandle.INSTANCE;
+    }
+
+    @Override
+    public void finishCreateTable(ConnectorSession session, ConnectorOutputTableHandle tableHandle, Collection<Slice> fragments) {}
+
+    @Override
+    public ConnectorInsertTableHandle beginInsert(ConnectorSession session, ConnectorTableHandle tableHandle)
+    {
+        return TestingHandle.INSTANCE;
+    }
+
+    @Override
+    public void finishInsert(ConnectorSession session, ConnectorInsertTableHandle insertHandle, Collection<Slice> fragments) {}
```

Comment:
It should be no-op in this particular case. It is required because i use it for `CTAS` testing.